spec/javascripts/views/story_view_spec.js

```diff
@@ -441,6 +441,13 @@ describe('StoryView', function() {
       expect(this.view.model.notes.last().isNew()).toBeTruthy();
     });
 
+    it("disables the add button if the text area is empty", function() {
```

Comment:
Don't you prefer to use like that?

```
describe("When the text area is empty", function() {
    it("Disables the add button", function() {
        // ...
    })
})
```

It's more readable and you could define more `it` inside your `when`. What do you think?

Reply:
good call, it'll be easier to make more tests later on.